pkg/config/validate.go

```diff
@@ -63,6 +64,21 @@ func ValidateClusterConfig(ctx context.Context, runtime runtimes.Runtime, config
 		return fmt.Errorf("The API Port can not be changed when using 'host' network")
 	}
 
+	// memory limits must have proper format
+	// if empty we don't care about errors in parsing
+	if config.ClusterCreateOpts.ServersMemory != "" {
+		if _, err := dockerunits.RAMInBytes(config.ClusterCreateOpts.ServersMemory); err != nil {
+			log.Errorf("Provided servers memory limit value is invalid")
+		}
+
+	}
+
+	if config.ClusterCreateOpts.AgentsMemory != "" {
+		if _, err := dockerunits.RAMInBytes(config.ClusterCreateOpts.AgentsMemory); err != nil {
+			log.Errorf("Provided agents memory limit value is invalid")
+		}
+	}
```

Comment:
I guess those two conditions should return an error as well, if it fails validation?